packages/framework/src/types/validator.types.ts

```diff
@@ -6,6 +6,7 @@ export type ValidateFunction<T = unknown> = AjvValidateFunction<T> | ((data: T)
 
 export type ValidationError = {
   path: string;
+  property: string;
```

Comment:
The mapping of the validation error to the schema object uses the property attribute with . and not /

I was wondering why we need the path @rifont ? should we maybe have only one of those (path/property) and where needed change one if those strings?

Reply:
You're absolutely right, we should use just one of these. My preference would be to modify the existing `path` property to use the `.`-separated syntax as I feel the name is better suited to describing a full path, where `property` is more singular.

Let's modify the validators to return `.`-separated syntax on the `path` property and remove the `property` entirely.